sass/cd/_cd-variables.scss

```diff
@@ -120,9 +120,23 @@ $cd-primary--dark: $cd-blue--dark;
 $cd-font: helvetica, arial, sans-serif;
 $cd-font--system: system-ui;
 
-// Web fonts.
-//@import url('https://fonts.googleapis.com/css?family=Mali:300,400,500,600,700&display=swap"');
-//$cd-font--webfont: 'Mali', cursive;
+// sass-lint:disable no-url-domains no-url-protocols
+
+// Multilingual
+// https://brand.unocha.org/d/xEPytAUjC3sH/visual-identity#/basics/fonts/multi-language-typefaces-1
+@import url('//fonts.googleapis.com/css2?family=Noto+Sans:wght@400;700&display=swap');
```

Comment:
An argument to self-host the fonts: https://wicki.io/posts/2020-11-goodbye-google-fonts/

Reply:
end of an era, eh?

Thanks for the link because otherwise I'd argue that people bouncing around between different OCHA sites would benefit from one cached resource.

We don't have to creep on this PR. It can be a separate ticket IMO.

Reply:
This is why the PR is here. We can get into it.

If I set this up for self-hosting, I would need to include the fonts in the repo, which I wanted to avoid doing because maintenance (maybe there is no maintenance? How often do fonts get updated?)

My first purpose here was to provide guidance for the fonts, and align with the brand guidelines.

If/when Roboto is used, that implementation will need to be in the subtheme so the base theme in this case serves only to provide some copy and paste rules, and take the thinking out of what font we should use. And because Roboto is optional, I also didn't want to include the font files.

As long as this is not blocking implementations from self-hosting fonts, I think it's a good first step. If we find we have a handful of implementations all opting for self-hosting, then maybe that's the evidence we should be more opinionated in the base theme.

What do you think?

Reply:
> How often do fonts get updated?

Roboto is pretty stable nowadays. I think in three years I did it once for DSR.

> My first purpose here was to provide guidance for the fonts, and align with the brand guidelines.

One thing that Google Fonts supports nowadays (and I see you implemented already) is the `font-display: swap` — a default that is far more important than the origin of the font in most cases. If we set up self-hosting, we would be keen to provide the `@font-face` definitions for folks as well, to encourage that they get loaded uniformly across properties.

> As long as this is not blocking implementations from self-hosting fonts, I think it's a good first step.

Agreed, it is better to provide a recommendation than to let people go nuts.

## UNOCHA-hosted?

Third option that potentially allows us to have our 🍰  and eat it too?

If we set up a simple static repo, e.g. a GitHub Pages with a CNAME like `fonts.unocha.org` I wonder if that allows at least the sites under the unocha.org domain to gain the benefits of cross-site caching, which also reduces our need to re-implement self-hosted fonts over and over again.

If we build self-hosting into the theme it might be overkill to offer such a service, but the idea popped into my head and I wanted to jot it down.

Reply:
I do enjoy cake.
I'm adding a note on the JIRA issue to track this.

Reply:
One more thought: our own non-unocha.org sites could still use this no different than google fonts. So implementation-wise it would be consistent across all our sites, with a potential performance benefit for unocha.org subdomains.

Reply:
> GitHub Pages with a CNAME

This might work, but first request will be slow as well and we depend on another third party for hosting it.

Reply:
Yup, well noted. The custom origin was just an idea for an experiment, and I don't think there's motion to implement atm.